Sources/Cells/TextFieldTableViewCell/TextFieldTableViewCell.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  Copyright © 2019 Rosberry. All rights reserved.
+//
+
+import UIKit
+
+final class TextFieldTableViewCell: BaseTableViewCell {
+
+    private var plugin: TextFieldPlugin?
+
+    // MARK: - Subviews
+
+    private lazy var textField: UITextField = {
+        let view = UITextField()
+        view.borderStyle = .roundedRect
+        view.delegate = self
+        view.addTarget(self, action: #selector(textFieldValueChanged), for: .editingChanged)
+        return view
+    }()
+
+    // MARK: - Lifecycle
+
+    override func setup() {
+        super.setup()
+        accessoryView = textField
+    }
+
+    override func configure(with plugin: Plugin) {
+        super.configure(with: plugin)
+        self.plugin = plugin as? TextFieldPlugin
+
+        if let plugin = self.plugin {
+            textField.placeholder = plugin.placeholder
+            textField.text = plugin.initialValue
+            textField.keyboardType = plugin.keyboardType
+        }
+    }
+
+    override func layoutSubviews() {
+        textField.bounds = CGRect(origin: .zero, size: .init(width: contentView.bounds.midX, height: 40))
+        super.layoutSubviews()
+    }
+
+    // MARK: - Actions
+
+    @objc private func textFieldValueChanged(_ sender: UITextField) {
+        plugin?.textFieldValueChanged(sender)
+    }
+}
+
+// MARK: - UITextFieldDelegate
+
+extension TextFieldTableViewCell: UITextFieldDelegate {
```

Comment:
Maybe we should have a handler for case when text field loses focus as well? It may be useful in case doing something on every text change would be an overkill (if it uses a lot of CPU or does a network request, for example); in this case developer will be able to put such task in `textFieldEditingEnded` handler or something like that.

Reply:
That's an optional one, I think; we can take care of it later.